applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element_volume_error_source.cpp

```diff
@@ -0,0 +1,1836 @@
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 2)[k] = 0.0;
+				}
+			}
+			pElement->GetGeometry()[0].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[1].FastGetSolutionStepValue(DISTANCE) =  1.0;
+			pElement->GetGeometry()[2].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[3].FastGetSolutionStepValue(DISTANCE) =  1.0;
+
+			// Compute RHS and LHS
+			Vector RHS = ZeroVector(16);
+			Matrix LHS = ZeroMatrix(16,16);
+
+			const auto& r_process_info = modelPart.GetProcessInfo();
+			pElement->Initialize(r_process_info); // Initialize the element to initialize the constitutive law
+			pElement->CalculateLocalSystem(LHS, RHS, r_process_info);
+			// Check the RHS values (the RHS is computed as the LHS x previous_solution,
+			// hence, it is assumed that if the RHS is correct, the LHS is correct as well)
+			std::cout << std::setprecision(12) << std::endl;
+        	
+			KRATOS_CHECK_NEAR(RHS(0), 3782.5533335, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(1), 2744.97890633, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(2), 3686.9679704, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(3), -5.52594881831, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(4), -3777.85819668, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(5), 237.470888418, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(6), -453.676393258, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(7), -4.00642325692, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(8), 10.2704695093, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(9), -3395.0486301, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(10), -469.951076124, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(11), -2.86019359732, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(12), 30.7635603322, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(13), 507.703002014, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(14), -4253.86133435, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(15), -4.37410099412, 1e-7);
+
+		}
+
+		// /** Checks the TwoFluidNavierStokes3D4N element
+	    //  * Checks the LHS and RHS for a cut element with surface tension
+	    //  */
+	    KRATOS_TEST_CASE_IN_SUITE(ElementTwoFluidNavierStokesSurfaceTension3D4NError, FluidDynamicsApplicationFastSuite)
+		{
+			Model current_model;
+			ModelPart& modelPart = current_model.CreateModelPart("Main");
+			modelPart.SetBufferSize(3);
+
+			// Variables addition
+			modelPart.AddNodalSolutionStepVariable(BODY_FORCE);
+			modelPart.AddNodalSolutionStepVariable(DENSITY);
+			modelPart.AddNodalSolutionStepVariable(DYNAMIC_VISCOSITY);
+			modelPart.AddNodalSolutionStepVariable(DYNAMIC_TAU);
+			modelPart.AddNodalSolutionStepVariable(PRESSURE);
+			modelPart.AddNodalSolutionStepVariable(VELOCITY);
+			modelPart.AddNodalSolutionStepVariable(MESH_VELOCITY);
+			modelPart.AddNodalSolutionStepVariable(DISTANCE);
+
+			// Process info creation
+			double delta_time = 0.1;
+			modelPart.GetProcessInfo().SetValue(DYNAMIC_TAU, 0.001);
+			modelPart.GetProcessInfo().SetValue(DELTA_TIME, delta_time);
+			modelPart.GetProcessInfo().SetValue(SURFACE_TENSION, true);

```

Comment:
Did you intentionally switch on the surface tension in here?

Reply:
Removed this test and added to test_two_fluid_navier_stokes_element.cpp